pandas/tests/plotting/test_datetimelike.py

```diff
@@ -419,6 +419,7 @@ def test_get_finder(self):
         assert conv.get_finder("A") == conv._annual_finder
         assert conv.get_finder("W") == conv._daily_finder
 
+    @pytest.mark.xfail  # I am not sure if this test is correct
```

Comment:
Can you expand on this? Perhaps open a new issue with the description of the bug, and reference it.

Reply:
Can you post images of the output for one of the failing tests, before and after your changes?

Reply:
@TomAugspurger yes, i feel the best way is to `xfail` the tests here and open up a new issue and do a follow-up PR to fix the tests, Because the main purpose is to test if the `finder` is being correctly used or not. And I honestly do not understand especially the second one in which there is a `vmin + 0.9`, I assume that's because of this wrong xaxis output, so has to manually add a constant to match the behavior.

Reply:
before my change:

![Screen Shot 2019-08-19 at 6 20 18 PM](https://user-images.githubusercontent.com/9269816/63282198-8f8d2200-c2ae-11e9-97ba-9743af0253f1.png)

after my change:

![Screen Shot 2019-08-19 at 6 18 23 PM](https://user-images.githubusercontent.com/9269816/63282220-99168a00-c2ae-11e9-8969-22de70fd3877.png)

@TomAugspurger

Reply:
Thanks... I think I agree with your assessment.

Can you open a new issue to discuss removing / updating those tests? And reference that in the skip / xfails?

Reply:
sure, i was thinking of doing it, but then i thought this PR would become a blocker for the issue if it is not merged because the test has to use the output of this PR. Probably I just overthink it ^^ thanks, will open and reference it. @TomAugspurger